spec/sitemap/generator_spec.rb

```diff
@@ -0,0 +1,15 @@
+# spec/sitemap/generator_spec.rb
+require 'spec_helper'
+
+describe SitemapGenerator::Interpreter do  
+  describe '.run' do
+    it 'does not raise an error' do
+      allow(SitemapGenerator::Sitemap).to receive(:ping_search_engines).and_return true
+      allow(SitemapGenerator::Sitemap).to receive(:create).and_yield
+
+      # Create some test data here with FactoryGirl
```

Comment:
This spec is passing when I make a change that keeps the rake task from running under development. I think it's because there's no test data to run against.

Can you create some test objects here? One or two of each major class that you're adding should do the trick. (E.g., 2 works, 1 collection)